### PR TITLE
[Merge] Master -> Dev

### PR DESCRIPTION
**Pull request by Pali-Pali**

### CODE REVIEW

**Comment by asandan8106** [comment]
nice , I like the color scheem so much but  please work on a code that would accept a keyword not matter how the question is structured  .

File: .vscode/launch.json
Comment (atlehang66) [inline .vscode/launch.json:1]: The bot has a good design. I recommend using terms like "skills" and "work" to help your chatbot understand user input.

File: index.html [hunk 1]
```diff
@@ -216,15 +224,109 @@ <h1 class="rainbow-text">Get in touch!</h1>
...
+                "What is your portfolio about?": "My portfolio showcases my work as an upcoming cobol mainframe developer. In my website you will find more information about myself an aspirations in life.",
+                "Can you tell me about your experience?": "I am currently an intern at capaciti.",
+                "Tell me about your education background": "I graduated from IIE Rosebank College in 2023 with a Bachelor of Information Technology in Business Systems. I previously graduated from the same university in 2020 with a Higher Certificate in Support Services. The National Senior Certificate, earned at Ekangala Comprehensive High School in 2018, marked the start of my academic career.",
+                "How can we reach you?": "You can reach out to me via the contact form on my website or email me directly at palesaclaudia2@gmail.com. I’d love to discuss your project!",
+                "Do you have any testimonials?": "Yes, I have received positive feedback from clients. You can find testimonials in the 'Testimonials' section of my portfolio."
+            },
```
Comment (atlehang66) [inline index.html:272]: This is where the keywords need to be implemented

File: .vscode/launch.json
Comment (Pali-Pali) [reply .vscode/launch.json:1]: I will work on that, thank you for your review

File: index.html (hunk 1 above)
Comment (Pali-Pali) [reply index.html:272]: I will work on adding the recommendation on that part

Comment (RolivhuwaMuzila) [reply index.html:272]: @Pali-Pali ...Oh yes, please use keywords instead @atlehang66 is right.

Comment (Xolanisyav04) [reply index.html:272]: BESIDE WHAT ATLEHANG MENTIONED, EVERYTHING ELSE IS NICE!

**Pull request merged**